Bindings/Python/setup.py

```diff
@@ -3,6 +3,12 @@
 import os
 import sys
 from setuptools import setup
+
+if os.path.isfile('../../bin/opensim-cmd.exe'): # local installation
```

Comment:
Add a comment here or below describing that providing the `bin` folder to `data_files` will copy them into the installation when using a `pip` install.